sonar-project.properties

```diff
@@ -15,9 +15,9 @@ sonar.tests=test
 
 sonar.cxx.jsonCompilationDatabase=build/compile_commands.json
 
-sonar.cxx.coverage.reportPath=build/reports/gcovr*.xml
-sonar.cxx.coverage.itReportPath=build/reports/gcovr*.xml
-sonar.cxx.coverage.overallReportPath=build/reports/gcovr*.xml
+#sonar.cxx.coverage.reportPath=build/reports/gcovr*.xml
+#sonar.cxx.coverage.itReportPath=build/reports/gcovr*.xml
+#sonar.cxx.coverage.overallReportPath=build/reports/gcovr*.xml
```

Comment:
why is this commented? 

even if no coverage is collected, these options may be enabled, in that case they do nothing